controllers/suite_test.go

```diff
@@ -65,9 +68,23 @@ var _ = BeforeSuite(func() {
 	ctx, cancel = context.WithCancel(context.TODO())
 
 	By("bootstrapping test environment")
+	k3sContainer, err := k3s.RunContainer(ctx,
+		testcontainers.WithImage("docker.io/rancher/k3s:v1.27.1-k3s1"),
```

Comment:
maybe from env variable?

Reply:
Yeah, sounds a good idea. Maybe keep this version as default value but allow a configuration using env var or something similar